openapis/swagger.yaml

```diff
@@ -0,0 +1,404 @@
+swagger: '2.0'
+info:
+  description: >-
+    Indexd is a prototype data indexing and tracking service. The code is open
+    source on [github](https://github.com/LabAdvComp/indexd)
+  version: 1.0.0
+  title: Indexd OpenAPI Specification
+  termsOfService: 'http://cdis.uchicago.edu/terms/'
+  contact:
+    email: cdis@uchicago.edu
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+host: data.braincommons.org
```

Comment:
Same here, Is it possible to have something generic here? Or use https://signpost.opensciencedatacloud.org/

Reply:
doh!  Good catch - I just pushed a fix.